cmd/agent-operator/agent-example-config.yaml

```diff
@@ -139,6 +232,7 @@ rules:
   resources:
   - nodes
   - nodes/proxy
+  - nodes/metrics
```

Comment:
This change, along with adding `/metrics/cadvisor`, should be reflected in the operator docs, but that is already covered by #923 so I'll let that get merged instead.